lib/modules/dosomething/dosomething_signup/dosomething_signup.install

```diff
@@ -625,3 +631,22 @@ function dosomething_signup_update_7022() {
     db_add_unique_key($table, 'uid-nid-run_nid', ['uid', 'nid', 'run_nid']);
   }
 }
+
+/**
+ * Adds campaign run nid column to the dosomething_signup table.
+ */
+function dosomething_signup_update_7023() {
```

Comment:
oh, this is update 7021 in `dev` 
https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_signup/dosomething_signup.install#L594